src/query/storages/delta/src/dal.rs

```diff
@@ -0,0 +1,426 @@
+// Copyright 2021 Datafuse Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// will be replaced by crate object_store_opendal later!
```

Comment:
Can we depend on git version of https://github.com/apache/incubator-opendal/tree/main/integrations/object_store?

Reply:
object_store_opendal deps on special version of opendal？

Reply:
Got it. Let me handle this part later.